Okta.AspNetCore/Okta.AspNetCore.csproj

```diff
@@ -30,7 +30,7 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
-		<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.3" />
+		<PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.8" />
```

Comment:
do we need to release a new version?

Reply:
Now that you mention it, we probably do.  I'll update the change log and version and let you know.  Thanks!